.gitignore

```diff
@@ -60,3 +60,5 @@ Basic_blink/.DS_Store
 EXTI/.settings/language.settings.xml
 
 EXTI/Debug/
+
+ADC/Debug/
```

Comment:
You can also simplify `ADC/Debug` and `EXTI/Debug` by only doing `./Debug`

Reply:
I thought I've done that. Gonna fix...

Reply:
I will take a look at it

Reply:
![Screen Shot 2020-04-16 at 19 34 29](https://user-images.githubusercontent.com/61191550/79513054-4ba71400-8019-11ea-8ffb-597227a0d4f0.png)

Reply:
there're more things over # Libraries line, but what I added is below #ADC/Debug line (I used the same from EXTI example)